plugins/cuda/torch_xla_cuda_plugin/__init__.py

```diff
@@ -10,4 +11,37 @@ def library_path(self) -> str:
 
   def physical_chip_count(self) -> int:
     # TODO: default to actual device count
-    return int(os.getenv('GPU_NUM_DEVICES', '1'))
+    return xu.getenv_as('GPU_NUM_DEVICES', int, 1)
```

Comment:
Perhaps irrelevant to this PR but just want to confirm that the `# TODO: default to actual device count` still holds, since `GPU_NUM_DEVICES` is not always set and the default value may not be 1.

Reply:
I see it's only used in `run_multiprocess`. So it sound using `GPU_NUM_DEVICES ` preserves the current behavior. Looks good to me then.

Reply:
Yeah, this is the same as the current behavior. Ideally this should check the PCI device IDs like we do for TPUs.